packages/bpk-component-map/examples.js

```diff
@@ -156,8 +158,8 @@ class StatefulBpkPriceMarker extends Component<
   render() {
     return (
       <StoryMap
-        zoom={12}
-        center={{ latitude: 35.661777, longitude: 139.704051 }}
```

Comment:
Changed the location to EDI so that it was easier to find the location and add markers to ensure they accurately mark the correct locations for the markers